props/prop_base.h

```diff
@@ -1409,6 +1409,19 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
       }
       return true;
     }
+    if (!strcmp(cmd, "twist")) {
```

Comment:
Not sure these should go in prop_base since lots of props don't have/use twists.

Reply:
ok sure. Moved to user prop instead.